src/ql/ir/compat/detail/cqasm_reader.cc

```diff
@@ -603,7 +603,7 @@ lqa::Analyzer ReaderImpl::build_analyzer() {
     a.register_function("operator!=", "bb", op_lxor_bb);
     for (const auto &gate : gateset) {
         a.register_instruction(gate->cq_insn);
-        a.register_instruction("skip", "i", false, false);
+        a.register_instruction("skip", "i", false, false);  // FIXME: why inside loop
```

Comment:
I propose to take it out of the loop, i.e. put it after the loop then. I also see no reason why it is in the loop, especially after analyzing libqasm on this.

Reply:
Fixed in commit 0cb35fbd